### DISCUSSION TITLE
运行exe以后无反应会闪退，点击回车则会弹出一段提示然后闪退

### DISCUSSION THREAD

**Discussion by LSLMMAX** [mikumifa/biliTickerBuy#133] [提问专区] [answered]
![IE}51 KAF1AG8H Z A @ G3](https://github.com/mikumifa/biliTickerBuy/assets/106451209/37bf8a83-37be-4af7-b77f-445ad8ea1a74)

**Comment by mikumifa** [comment]
最新版吗

**Reply by LSLMMAX** [reply]
是最新版

**Comment by mikumifa** [comment]
@Amorter

**Comment by Amorter** [comment]
什么版本

**Reply by LSLMMAX** [reply]
最新版

**Reply by Amorter** [reply]
> 最新版

确定版本号是2.8.2吗，可以加下我qq吗1145252643

**Comment by vovoRRR** [comment]
我也是这个问题，闪一下就关掉了

**Reply by Amorter** [reply]
> 我也是这个问题，闪一下就关掉了

可以试一下dx修复工具吗，如果还是有问题麻烦告诉我一下

**Reply by vovoRRR** [reply]
修复过了，还是一样
![11](https://github.com/mikumifa/biliTickerBuy/assets/167942772/98b2bd04-e5d7-4413-8c63-8363aa9ad71e)

**Reply by mikumifa** [reply]
> 修复过了，还是一样
> ![11](https://github.com/mikumifa/biliTickerBuy/assets/167942772/98b2bd04-e5d7-4413-8c63-8363aa9ad71e)
> 
> 

看一下最新版

**Comment by Amorter** [comment, marked as answer]
不支持的平台只能暂时取消本地验证码功能了，如果需要的话自行升级win11或者较新的win10吧，系统缺什么dll真的不清楚，麻了

**Reply by LSLMMAX** [reply]
目前v2.8.4-patch.2能用了，只是本地验证码功能用不了
![image](https://github.com/mikumifa/biliTickerBuy/assets/106451209/ecfcad59-ca17-4c03-ab94-cf065d9cad76)

**Reply by mikumifa** [reply]
是的，你的电脑不支持就没有这个功能，想要这个功能的话升级一下系统
